website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -292,7 +292,10 @@ The examples described in this section demonstrate how to export a service named
     $ kubectl --context $CLUSTER2_CONTEXT apply --filename intention.yaml
     ```
 
-1. Add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods before deploying the workload so that the services in `cluster-01` can dial `backend` in `cluster-02`. To dial the upstream service from an application, configure the application so that that requests are sent to the correct DNS name as specified in [Service Virtual IP Lookups](/docs/discovery/dns#service-virtual-ip-lookups). In the following example, the annotation that allows the workload to join the mesh and the configuration provided to the workload that enables the workload to dial the upstream service using the correct DNS name is highlighted. 
+1. Add the `"consul.hashicorp.com/connect-inject": "true"` annotation to your service's pods before deploying the workload so that the services in `cluster-01` can dial `backend` in `cluster-02`. 
+To dial the upstream service from an application, configure the application so that that requests are sent to the correct DNS name as specified in [Service Virtual IP Lookups](/docs/discovery/dns#service-virtual-ip-lookups). 
+In the following example, the annotation that allows the workload to join the mesh and the configuration provided to the workload that enables the workload to dial the upstream service using the correct DNS name is highlighted. 
+[Service Virtual IP Lookups for Consul Enterprise](/docs/discovery/dns#service-virtual-ip-lookups-for-consul-enterprise) details how you would similarly format a DNS name including partitions and namespaces.
 
     <CodeBlockConfig filename="frontend.yaml" highlight="36,51">
```

Comment:
not blocking this PR since you didn't make changes to this code block, but I noticed in the rendering that this code block isn't getting rendered properly.

Reply:
Good catch! Turns out that component doesn't like that I formatted the bullet on multiple lines 😒 .